src/browser/DeviceProxy.js

```diff
@@ -37,8 +37,16 @@ function getBrowserInfo (getModel) {
     var returnVal = '';
     var offset;
 
-    if ((offset = userAgent.indexOf('Edge')) !== -1) {
-        returnVal = (getModel) ? 'Edge' : userAgent.substring(offset + 5);
+    if ((offset = userAgent.indexOf('Edg')) !== -1) {
+        if (getModel) {
+            returnVal = 'Edge';
+        } else {
+            if ((offset = userAgent.indexOf('Edge')) !== -1) {
```

Comment:
This won't work, as the offset might be set to `-1` again and line 47 would return `userAgent.substring(-1 + 4);`